BuildGuide.md

```diff
@@ -0,0 +1,126 @@
+# Build Guide
+
+***Disclaimer***
+
+While this guide is meant to give Iron180 owners a comprehensive overview of how to build a 180, please keep in mind that there are many variations/alternatives to some of the processes described here. At the end of the day, some of these choices come to preference, so do not worry if you see other individuals building boards in a different way. That being said, if you are more of a visual person and would rather see videos, we have linked a few links to build streams below.
+
+
+## Table of contents
+
+* [Parts check and PCB](#parts-check-and-pcb)
+* [Dissasembly](#dissasembly)
+* [Bumpons, Daughterboard, and Gaskets](#bumpons-daughterboard-and-gaskets)
+* [Stabilizers and Layout](#stabilizers-and-layout)
+* [Switches](#switches)
+* [Soldering](#soldering)
+* [Reassembly](#reassembly)
+
+## Parts check and PCB
+
+Before building the iron 180, it is important to take a few precautionary steps. Before you take anything apart or start building, we recommend that you double-check the [parts list](Iron180PartsList.md) to ensure that nothing is missing.
+
+You can also refer to the following photo as a parts reference throughout the build.
+
+![Parts](Photos/Iron180Photos/parts_mod.jpg)
+
+### Screws
+
+There are multiple options when it comes to screws for the back of your Iron. There are duplicate sets for color preferences as well as short and long screws. The short screws are the standard option that will not be visible when tightened, whereas the longer option allows the screws to sit flush with the bottom of the board, giving it a unique aesthetic appeal. You will always use four short screws (pink underline) for the front of the board.
+
+Note: If you decide to use the longer screws (green underline), you will need to be used with the included spacers before tightening. Failure to do so might result in damage to your board.
+
+### Checking your PCB
+
+With the advances in modern technology, it is very tempting/easy to skip this step, but we highly recommend that you take the time to test your PCB. Spending a few hours building your board only to find out after that something is wrong and you need to disassemble everything is going to be a wrong time. Unless you have the proper equipment to desolder, it will take you a lot longer to disassemble than reassemble.
```

Comment:
maybe `With more modern manufacturing processes` instead of `With the advances in modern technology`; the technology hasnt really changed

Reply:
modified